Microsoft.Health.Fhir.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28729.10
+# Visual Studio 15
```

Comment:
Downgrading?

Reply:
I don' think this affects anything, but probably should revert. We can't go back :P